packages/pwa/app/hooks/use-add-to-cart-modal.js

```diff
@@ -139,7 +140,7 @@ export const AddToCartModal = () => {
                                         {!!lineItemPrice &&
                                             intl.formatNumber(lineItemPrice, {
                                                 style: 'currency',
-                                                currency: currency || DEFAULT_CURRENCY
+                                                currency: currency || site.defaultCurrency
```

Comment:
Will currency ever be undefined for a basket? If it's always defined we might not require the use of the `useSite` hook and a fallback currency value for this use of `formatNumber`.

Follow up question: Are we updating the basket if the currency changes? We might want to make sure we are doing that as to not show a basket in the wrong currency after a use selects a new locale.

Reply:
The basket currency will not be defined when a new user gets on the app for the first time, but it will be created right away. In this modal, I'd say we can safely remove the fallback currency because the modal only pops up when an item is added to cart. At this certian point of time, the currency has always been defined. 

For follow-up question: Yes, the basket is updated with currency changes.